docs/source/en/tasks/object_detection.mdx

```diff
@@ -0,0 +1,585 @@
+<!--Copyright 2022 The HuggingFace Team. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+the License. You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+specific language governing permissions and limitations under the License.
+-->
+
+# Object detection
+
+[[open-in-colab]]
+
+Object detection is a computer vision task of detecting instances of semantic objects that belong to a particular
+class (such as humans, buildings, or cars) in parts of an image. Object detection models receive an image as input and output
+coordinates of the bounding boxes and associated labels of the detected objects. An image can contain multiple objects,
+each with its own bounding box and a label (e.g. it can have a car and a building).
+be present in different parts of an image (e.g. the image can have several cars).
+This task is commonly used in autonomous driving for detecting things like pedestrians, road signs, and traffic lights.
+Other applications include counting objects in images, image search, and more.
+
+ <Tip>
+Check out the <a href="https://huggingface.co/tasks/object-detection">object detection</a> task page to learn about use cases,
+models, metrics, and datasets associated with this task.
+ </Tip>
+
+In this guide, you will learn how to:
+
+ 1. Finetune [DETR](https://huggingface.co/docs/transformers/model_doc/detr), a model that combines a convolutional
+ backbone with an encoder-decoder Transformer, on the [CPPE-5](https://huggingface.co/datasets/cppe-5)
+ dataset.
+ 2. Use your finetuned model for inference.
+
+Before you begin, make sure you have all the necessary libraries installed:
+
+```bash
+pip install -q datasets transformers evaluate timm albumentations
+```
+
+In this guide you'll use `datasets` to load a dataset from the Hugging Face Hub, `transformers` to train your model,
+and `albumentations` to augment the data. `timm` is currently required to load a convolutional backbone for the DETR model.
+
+We encourage you to share your model with the community. Log in to your Hugging Face account to upload it to Hub.
+When prompted, enter your token to log in:
+
+```py
+>>> from huggingface_hub import notebook_login
+
+>>> notebook_login()
+```
+
+## Load the CPPE-5 dataset
+
+In this guide, you'll use the [CPPE-5 dataset](https://huggingface.co/datasets/cppe-5) that contains images with
+annotations identifying medical personal protective equipment (PPE) in the context of the COVID-19 pandemic.
```

Comment:
Please include a guide on how people can use their own data. Using a HuggingFace dataset is nice, but the first question people always come up with is "but how I can use my own custom dataset?".

Here I'd recommend adding a link to ImageFolder + metadata, which allows people to turn their own image captions into a 🤗 Dataset: https://huggingface.co/docs/datasets/v2.8.0/en/image_dataset#image-captioning

Reply:
This is out of scope of the tasks guide. None of the other one does that.

Reply:
If this is a common question, perhaps, this can be a note or a tip in this doc? https://huggingface.co/docs/transformers/training
Something along the lines "In the Transformers documentation we use datasets from the Hugging Face Hub. If you would like to use your own custom data, check out the Datasets documentation on how to create a dataset" ?

Reply:
That would be a good addition IMO.